components/content/GiscusComments.vue

```diff
@@ -0,0 +1,48 @@
+<script setup>
+function fixupChineseLocale(locale) {
+    if (locale == "zh-hans") {
+        return "zh-CN";
+    } else if (locale == "zh-hant") {
+        return "zh-TW";
+    } else {
+        return locale;
+    }
+}
+</script>
+
+<template>
+    <Giscus
+        id="comments"
+        repo="haroohie-club/HaroohieSite"
+        repoId="R_kgDOG5Llsw"
+        category="Announcements"
+        categoryId="DIC_kwDOG5Lls84Cc20X"
+        mapping="specific"
+        :term="slug"
+        strict="1"
+        reactionsEnabled="1"
+        emitMetadata="0"
+        inputPosition="top"
+        theme="light"
+        :lang="fixupChineseLocale(lang)"
+    />
```

Comment:
this is probably fine but maybe wrap this in a div because putting elements directly in a <template> can cause some wacky stuff depending on the elements of the sub template

Reply:
will do